File: javascript/1134_tipoDeCombustivel.js
var readline = require("readline");
var rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let alcool = 0;
let gasolina = 0;
let diesel = 0;

async function recursiveAsyncReadLine() {
  rl.question("", (answer) => {
    if (answer == 4) return rl.close();
    else if (answer == 1) alcool++;
    else if (answer == 2) gasolina++;
    else if (answer == 3) diesel++;
    recursiveAsyncReadLine();
  });
}

recursiveAsyncReadLine().then(
  console.log(`
    MUITO OBRIGADO
    Alcool: ${alcool}
    Gasolina: ${gasolina}
    Diesel: ${diesel}`)
);

File: python3/1134_tipoDeCombustivel.py
alcool = 0
gasolina = 0
diesel = 0

while True:
    combustivel = int(input())

    if(combustivel == 1):
        alcool += 1
    elif(combustivel == 2):
        gasolina += 1
    elif(combustivel == 3):
        diesel += 1
    elif(combustivel == 4):
        break

print("MUITO OBRIGADO")
print("Alcool:", alcool)
print("Gasolina:", gasolina)
print("Diesel:", diesel)
